src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,83 @@
+.wrapper {
+    padding-bottom: 24px;
+    margin-top: 64px;
+    max-width: 186px;
+    margin-left: 16px;
+    min-height: 310px;
+    
+}
+
+.list {
+    padding: 0;
+    list-style-type: none;
+}
+
+.footerItem {
+    color: white;
+    margin-bottom: 32px;
+    font-family: "InterRegular";
+    font-size: 24px;
+    line-height: 24px;
+}
+
+.copyright {
+    margin-top: 48px;
+    color:white;
+    font-family: "InterRegular";
+    font-size: 14px;
+    line-height: 14px;
+}
+
+.link, .link:visited {
+    text-decoration: none;
+    color: white;
+}
+
+.link:hover {
+    text-decoration: underline;
+}
+
+@media (min-width: 768px) {
+    .wrapper {
+        display: flex;
+        flex-direction: row;
```

Comment:
the default `flex-direction` is row, you could remove this line

Reply:
fixed